src/main/java/aqua/MainApp.java

```diff
@@ -0,0 +1,20 @@
+package aqua;
+
+import aqua.graphic.MainWindow;
+import aqua.manager.LogicManager;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+
+/** Main application class. */
+public class MainApp extends Application {
+    @Override
+    public void start(Stage primaryStage) {
+        MainWindow window = new MainWindow(new LogicManager());
```

Comment:
I like how you separated the definition of the GUI components into another class. This would make it easier to for future developments. Also, it enforces OOP. Good job!